src/frontend/apps/e2e/__tests__/app-impress/doc-search.spec.ts

```diff
@@ -1,20 +1,21 @@
 import { expect, test } from '@playwright/test';
-import { DateTime } from 'luxon';
 
 import { createDoc, verifyDocName } from './common';
 
-type SmallDoc = {
-  id: string;
-  title: string;
-  updated_at: string;
-};
-
 test.beforeEach(async ({ page }) => {
   await page.goto('/');
 });
 
 test.describe('Document search', () => {
-  test('it checks all elements are visible', async ({ page }) => {
+  test('it checks all elements are visible', async ({ page, browserName }) => {
```

Comment:
maybe merge the two tests because we are duplicating the same test actions a bit, I find it a bit of a shame to take time for nothing. I don't have a strong opinion. Just a note

Reply:
Done